src/app/public/src/modules/link/link.directive.ts

```diff
@@ -0,0 +1,31 @@
+import { Directive, ElementRef, HostListener, Input, Renderer2 } from '@angular/core';
+
+import { StacheNavService } from '../nav';
+
+@Directive({
+  selector: '[stacheRouterLink]'
+})
+export class StacheRouterLinkDirective {
+  @Input()
+  public stacheRouterLink: string;
+
+  @Input()
+  public fragment: string;
+
+  constructor(
+    private navService: StacheNavService,
+    private el: ElementRef,
+    private renderer: Renderer2
+  ) {
+    this.renderer.setStyle(this.el.nativeElement, 'cursor', 'pointer');
+   }
```

Comment:
I think you have an extra space before the closing curly bracket, here.

Reply:
Doh!